.github/workflows/tf-apply.yaml

```diff
@@ -19,6 +19,9 @@ on:
       aws_oidc_role_arn:
         description: "AWS OIDC IAM role to assume"
         type: string
+      gh_artifact_path:
```

Comment:
Shall we precise something like: "If value is not set, artifacts are not downloaded" or am I too paranoid 🤣 It might be confusing otherwise idk

Reply:
fair call, I'll add some extra info